vapi/Flatpak-1.0.metadata

```diff
@@ -0,0 +1,2 @@
+Transaction.operation_done.commit nullable
+TransactionErrorDetails.fatal name="non_fatal"
```

Comment:
Can you also add `TransactionErrorDetails cprefix="FLATPAK_TRANSACTION_ERROR_DETAILS_"`

Reply:
Oooh yes, that's nicer, thanks!